examples/seq2seq/finetune_trainer.py

```diff
@@ -266,17 +260,19 @@ def main():
     use_task_specific_params(model, data_args.task)
 
     # set num_beams for evaluation
-    if data_args.eval_beams is not None:
-        model.config.num_beams = data_args.eval_beams
-    assert model.config.num_beams >= 1, f"got eval_beams={model.config.num_beams}. Need an integer >= 1"
+    if data_args.eval_beams is None:
+        data_args.eval_beams = model.config.num_beams
+    assert data_args.eval_beams >= 1, f"got eval_beams={model.config.num_beams}. Need an integer >= 1"  # TODO: remove?
```

Comment:
IMO we can remove this `assert`, default is `None` and highly unlikely that someone will set it to 0 or negative